docs/developer/10-scm-plugins.md

```diff
@@ -23,6 +23,34 @@ Alternately, you can use the [Rundeck API - SCM](/api/rundeck-api.md#scm).
 SCM Configuration for a project is _not_ stored in the `project.properties`
 configuration contents.
 
+## Tuning 
+
+The following properties control how Process Automation interacts with external version control systems, such as Git or Subversion, to fetch and manage project definitions and job configurations. Here's a breakdown of each property:
+
+1. `rundeck.scmLoader.delay=0`
+
+   - **Purpose**: This property sets the delay (in seconds) before the SCM loader begins its first execution after Rundeck starts.
+   - **Default Value**: 0 (i.e., no initial delay)
+   - **Usage**: You can adjust this value if you want to introduce a delay before Rundeck starts loading projects and job configurations from the SCM repository after the Rundeck service is launched. This can be helpful to ensure that other components or services are fully ready before Rundeck begins its synchronization.
+
+2. `rundeck.scmLoader.interval=20`
+
+   - **Purpose**: This property determines the interval (in seconds) at which the SCM loader will check the SCM repository for changes and update the Rundeck project accordingly.
+   - **Default Value**: 20 seconds
+   - **Usage**: You can modify this value to specify how often Rundeck should poll the SCM repository for updates. A shorter interval means more frequent checks for changes, while a longer interval reduces the frequency of checks. Choose an appropriate value based on your SCM repository's update frequency and performance considerations.
+
+3. `rundeck.scmLoader.init.delay=10`
+
+   - **Purpose**: This property sets an initial delay (in seconds) before the SCM loader performs its first check after Rundeck has started.
+   - **Default Value**: 10 seconds
+   - **Usage**: Similar to the first property, this one introduces a delay, but it affects the initial check for updates after Rundeck has started. You might use this delay to allow other components or services to complete their startup procedures before SCM synchronization begins.
```

Comment:
Instead of saying "first property" use `rundeck.scmLoader.delay` in case we change what's "first" in the future.

Reply:
You didn't change anything here